code/modules/mob/living/carbon/human/examine_vr.dm

```diff
@@ -0,0 +1,22 @@
+/mob/living/carbon/human/examine/proc/examine_weight(mob/user)
+
```

Comment:
Declare /var/msg (could be named anything for convienence) and change the += in this file to =.

Basically this file needs to set some message string and return that up to examine.